### PR TITLE
Added Player HUD

### PR DESCRIPTION
**Pull request by NamStalker**
Added the player HUD with player information, health bar, main and side arms with ammo counters. Refactored to reduce size of the mod, and reduce some of the calculations. Added Weapons class to handle weapon properties and resources. Updated the common.json to reflect resolution changes, this was discovered as I use a 4k monitor, and realized setting the game to 1920x1080 was the only way to get the display working initially.

### CODE REVIEW

**Comment by NamStalker** [comment]
issues #3

File: GGO.Common/Draw.cs [hunk 1]
```diff
@@ -28,33 +29,54 @@ public static void Icon(Configuration Config, string ImageFile, Point Position)
...
-                HealthSize = Config.PlayerHealthSize;
-            }
+            Size InfoSize = isPlayer ? Config.PlayerInfoSize : Config.SquadInfoSize;
+            Size HealthSize = isPlayer ? Config.PlayerHealthSize : Config.SquadHealthSize;
+            Size HealthPos = isPlayer ? Config.PlayerHealthPos : Config.SquadHealthPos;
+            string iconText = isPlayer ? "Player" : "Squad" + SquadCount;
+            Single textScale = isPlayer ? 0.4f : 0.3f;
+
+            // Draw the player icon
+            if(Character.IsAlive)
```
Comment (justalemon) [inline GGO.Common/Draw.cs]: 2 things:
* There should be an space after the word if.
* {} Should be used

File: GGO.Common/Draw.cs [hunk 2]
```diff
@@ -69,21 +91,61 @@ public static void PedInfo(Configuration Config, Ped Character, Point Position,
...
+            List<Weapons.Type> NoDraw = new List<Weapons.Type>() { Weapons.Type.Banned, Weapons.Type.Melee };
+            NoDraw.Add(isOffHand ? Weapons.Type.Main : Weapons.Type.Sidearm);
+
+            // Do not display main hand weapons for these types.
+            if (NoDraw.Contains(Weapons.CurrentWeaponType))
+            {
+                Icon(Config, Common.Image.ResourceToPNG(Properties.Resources.NoWeapon, "NoWeapon" + hand), HandPosition);
+                Icon(Config, Common.Image.ResourceToPNG(Properties.Resources.NoWeapon, "NoAmmo" + hand), AmmoPosition);
+                // Only continue on to draw weapon if it is melee and this is the off hand.
+                if(!(Weapons.CurrentWeaponType == Weapons.Type.Melee && isOffHand))
```
Comment (justalemon) [inline GGO.Common/Draw.cs]: Same here

**Review by justalemon — CHANGES_REQUESTED** [review]
I have not tested it yet, but a change needs to be made

File: GGO.Common/Draw.cs (hunk 2 above)
Comment (SleepyNatDev) [reply GGO.Common/Draw.cs]: Changes have been made.

**Review by justalemon — APPROVED** [review]

**Pull request merged**